docs/SetUpDevEnv.md

```diff
@@ -0,0 +1,9 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Set up your development environment
+
+### Install Visual Studio
+1. Install Visual Studio 2017 from [here](https://visualstudio.microsoft.com/vs/)
+   - Choose ".NET desktop development" option
+   - Add ".NET Framework 4.7.1 development tools"
```

Comment:
>4.7.1 [](start = 25, length = 5)

AIWin has components that require .NET 4.7.1. Is there anything in axe-windows that requires that version by itself? For example, could axe-windows use .NET 4.5.2 or 4.6? Not necessarily saying that we want to change it now, but maybe in the future

Reply:
That is a safely deferable decision.